### DISCUSSION TITLE
helm chart postgrsql configuration

### DISCUSSION THREAD

**Discussion by tirelibirefe** [apache/skywalking#8375] [Q&A]
Hello,
I would like to install Skywalking by using helm chart on Kubernetes. I would like to use Postgresql instead of ElasticSearch but I couldn't have found any reference for postgresql connection. How can I configure Postgresql connection in values.yaml ?

Thanks & Regrads

**Comment by wu-sheng** [comment]
Read the doc first, `helm` is only a deployment tool, most features are in the doc already.

https://skywalking.apache.org/docs/main/v8.9.1/en/setup/backend/backend-storage/#postgresql

**Reply by tirelibirefe** [reply]
I already read them.
It is not for Helm.
I need to reference for Helm values.yaml file.

**Reply by wu-sheng** [reply]
This is only a system environment setting.

**Reply by tirelibirefe** [reply]
ok good but how/where to use them?
I use Helm to deploy Skywalking.
Where/how to define Postgresql parameters in Helm values.yaml when I want to use Postgresql as backend?

**Reply by wu-sheng** [reply]
Take this as an example. https://github.com/apache/skywalking/blob/master/test/e2e-v2/cases/istio/metrics/e2e.yaml#L69

**Reply by tirelibirefe** [reply]
is there a Potgresql defined as backend?

**Reply by wu-sheng** [reply]
I think you are still not following the document. Read the introduction to understand what YAML means, https://skywalking.apache.org/docs/main/v8.9.1/en/setup/backend/backend-setup/#applicationyml. Then follow the example of why using system env in the Helm.

I don't think you need more document
